readme.md

```diff
@@ -65,7 +70,7 @@ Describe the purpose of views and models in FE JS.
 
 Your Answer:
 ```text
-
+Views and models in FE JS are to display on the client side the information.  
```

Comment:
Looking for more depth here (what's the difference between the two?). Check out the solution branch for more info.